evaluations/README.md

```diff
@@ -21,4 +22,7 @@ Some other relevant files to look at:
 
 * Motion Planning:
     - `theseus/utils/examples/motion_planning/motion_planner.py`: Puts together optimization layer.
-    - `examples/motion_planning_2d.py`: Implements outer loop.
\ No newline at end of file
+    - `examples/motion_planning_2d.py`: Implements outer loop.
+
+* Homography Estimation:
+    - `examples/homography_estimation.py`: Puts together optimization layer and implements outer loop.
```

Comment:
The description here isn't informative. What's would we like to communicate here? If we just want to have a pointer to this example then we can add the link to it in the autodiff_cost_function_ablation.sh description above.

Reply:
I just followed the description of other examples https://github.com/facebookresearch/theseus/blob/2ad577aa6a912db037dd7eb4ee76aa37bd1935e0/evaluations/README.md?plain=1#L18.

Reply:
Those are just extra links for people to look at. I didn't think we had to be too explicit, the main point of this readme file is to describe what the bash script do, not to describe all of our python scripts; those already have some comments describing what they do.